src/prognostic_equations/zero_tendency.jl

```diff
@@ -0,0 +1,37 @@
+###
+### Zero grid-scale or subgrid-scale tendencies
+###
+
+function zero_tendency!(Yₜ, Y, p, t, colidx, ::NoGridScaleTendency, _)
```

Comment:
Why dont we also need the `nothing` version?

Reply:
I forgot to add it, thanks!